src/Model/Booking.php

```diff
@@ -420,7 +414,7 @@ public function bookingLinkUrl(): string {
 	 * @return bool
 	 */
 	public function isCancelled(): bool {
-		return ( $this->post_status == 'canceled');
+		return $this->post->post_status === 'canceled';
```

Comment:
Also wir benutzen relativ häufig an verschiedenen Stellen Magic Methods, um die Properties von Post abzufragen. Ich bin mir unsicher ob, wenn wir das hier ändern es nicht auch woanders ändern müssen oder ob wir das nicht vielleicht einfach so lassen. Ich stelle mir so einen Mix potenziell verwirrend vor.

Reply:
Meinst du überhaupt die Einführung der `is*`-Methoden oder nur die spezielle Abfrage an dieser Stelle nicht über die Magic-Method?